example/src/main/java/ably/io/tutorial/ExampleActivity.java

```diff
@@ -34,12 +39,28 @@ protected void onCreate(Bundle savedInstanceState) {
         } catch (AblyException e) {
             e.printStackTrace();
         }
+        findViewById(R.id.btPublish).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                /* Get message content from EditText */
+                String message = messageEditText.getEditableText().toString();
+                /* Check if message is not empty */
+                if (!TextUtils.isEmpty(message)) {
+                    try {
+                        Log.v(getClass().getCanonicalName(), "Publishing message...");
```

Comment:
As this is an Android demo, I assume the user will never see this message.  Should we not instead show this in the UI?